extensions/data-plane/data-plane-google-storage/src/main/java/org/eclipse/edc/connector/dataplane/gcp/storage/GcsDataSink.java

```diff
@@ -52,10 +53,14 @@ protected StreamResult<Object> transferParts(List<DataSource.Part> parts) {
             } catch (IOException e) {
                 monitor.severe("Cannot open the input part", e);
                 monitor.severe(e.toString());
-                return StreamResult.error("An error");
+                return StreamResult.error("Cannot open the input part");
+            } catch (EdcException e) {
+                monitor.severe("Cannot open the source blob", e);
+                monitor.severe(e.toString());
+                return StreamResult.error("Cannot open the source blob");
```

Comment:
could this catch branch need to be tested

Reply:
The catch block was added to handle the blob existence check in the sink (now removed), not used anymore: removing.

Reply:
Removed